app/models/user.rb

```diff
@@ -88,6 +88,10 @@ def self.find_for_oauth(auth, signed_in_resource = nil)
     user
   end
 
+  def self.leaders(after = Time.new(0))
+    User.all.sort_by { |user| -user.points }
```

Comment:
This is slower compare to its alternative:

```
User.all.sort_by { |user| user.points }.reverse
```

See: http://stackoverflow.com/questions/2642182/sorting-an-array-in-descending-order-in-ruby